clients/src/test/java/org/apache/kafka/clients/consumer/internals/MembershipManagerImplTest.java

```diff
@@ -1931,7 +1931,8 @@ public void testStaleMemberWaitsForCallbackToRejoinWhenTimerReset() {
         membershipManager.onHeartbeatRequestSent();
 
         assertEquals(MemberState.STALE, membershipManager.state());
-        verify(backgroundEventHandler).add(any(ConsumerRebalanceListenerCallbackNeededEvent.class));
+        assertFalse(backgroundEventQueue.isEmpty());
```

Comment:
Since ```backgroundEventHandler``` is an actual instance of an object, we cannot ```verify()``` on it. So in order to still test what it was trying to test, the two assert statements must be there instead

Reply:
does changing `backgroundEventHandler` to a mock break a lot of tests?

Reply:
Yeah, breaks 16/81, all test failures are on asserting an empty queue or number of events in the queue